root/elections/ElectionTable/ElectionTableRows.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l, lp} = require('../../static/scripts/common/i18n');
+const EditorLink = require('../../static/scripts/common/components/EditorLink');
+const formatUserDate = require('../../utility/formatUserDate');
+import {votesVisible} from '../../utility/voting';
+
+type PropsT = {
+  +elections: $ReadOnlyArray<AutoEditorElectionT>,
+};
+
+const ElectionTableRows = ({elections}: PropsT) => (
+  elections.map((election, index) => (
+    <tr className={index % 2 ? 'even' : 'odd'} key={election.id}>
+      <td><EditorLink editor={election.candidate} /></td>
+      <td>{lp(election.status_name_short, 'autoeditor election status (short)')}</td>
+      <td>{formatUserDate($c.user, election.propose_time)}</td>
+      <td>{election.close_time ? formatUserDate($c.user, election.close_time) : '-'}</td>
```

Comment:
Same here, just displays `%c` for me. (I have a custom date format set in /account/preferences if that's related.)

Reply:
This is because moment-strftime does not support %c or %x/%X. How do you want to deal with it?

The easiest way I can see is to add a json comversion sub to convert these formats to moment compatible formats.

Reply:
Not sure I see a reason moment-strftime can't support those, since moment has localized format options. If so I'd fork moment-strftime and submit a patch upstream, then just use the fork in package.json for now.

Reply:
I actually ran into the same issue in the area search results PR (displays "%c" for the last updated time). So if you'd rather I look into it, I can tomorrow, but otherwise feel free. :) If I'm wrong and moment-strftime can't be easily patched, then a conversion method sounds fine, though I'd probably do the conversion in the JavaScript instead.